tests/ui/rfcs/impl-trait/higher-ranked-regions-diag.rs

```diff
@@ -0,0 +1,25 @@
+// Regression test for #97099.
+// This was an ICE because `impl Sized` captures the lifetime 'a.
+
+// check-fail
```

Comment:
@oli-obk Oli did you forgor to update these UI test directives to use `//@`?

Reply:
`check-fail` should be the default for these tests anyways, so we could remove them rather than use `//@`